tools/train.py

```diff
@@ -98,6 +98,17 @@ def main():
     args = parse_args()
 
     cfg = Config.fromfile(args.config)
+
+    # update data root according to environment variable
+    if os.environ.get('MMDET_DATASETS', None) is not None:
```

Comment:
This logic needs to be written as a separate function, because `test.py` and`browse_dataset.py` also need.

Reply:
After that, add unit tests to ensure correctness. Thank you !

Reply:
> After that, add unit tests to ensure correctness. Thank you !

@hhaAndroid 😸 Okay~~